sys-libs/libatomic_ops/libatomic_ops-7.6.2.recipe

```diff
@@ -0,0 +1,78 @@
+SUMMARY="An atomic memory update operations portable implementation"
+DESCRIPTION="Semi-portable access to hardware-provided atomic memory update \
+operations on a number architectures."
+HOMEPAGE="https://github.com/ivmai/libatomic_ops"
+COPYRIGHT="1991-1994 by Xerox Corporation.  All rights reserved.
+	1996-1999 by Silicon Graphics.  All rights reserved.
+	2003-2011 Hewlett-Packard Development Company, L.P.
+	2005,2007 Thiemo Seufer
+	2007 by NEC LE-IT
+	2016 Marek Vasut"
+LICENSE="MIT
+	GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/ivmai/libatomic_ops/releases/download/v$portVersion/libatomic_ops-$portVersion.tar.gz"
+CHECKSUM_SHA256="219724edad3d580d4d37b22e1d7cb52f0006d282d26a9b8681b560a625142ee6"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libatomic_ops$secondaryArchSuffix = $portVersion
+	lib:libatomic_ops$secondaryArchSuffix = 1.1.1 compat >= 1
+	lib:libatomic_ops_gpl$secondaryArchSuffix = 1.1.2 compat >= 1
```

Comment:
You might wish to define 4 custom variables (like `lib{ops,gpl}Version{,Compat}`) and use them later. Here's a suggestion:
```diff
 SECONDARY_ARCHITECTURES="x86"
+
+libopsVersion="1.1.1"
+libgplVersion="1.1.2"
+
+libopsVersionCompat="$libopsVersion compat >= ${libopsVersion%%.*}"
+libgplVersionCompat="$libgplVersion compat >= ${libgplVersion%%.*}"
+
 PROVIDES="
 	libatomic_ops$secondaryArchSuffix = $portVersion
-	lib:libatomic_ops$secondaryArchSuffix = 1.1.1 compat >= 1
+	lib:libatomic_ops$secondaryArchSuffix = $libopsVersionCompat
-	lib:libatomic_ops_gpl$secondaryArchSuffix = 1.1.2 compat >= 1
+	lib:libatomic_ops_gpl$secondaryArchSuffix = $libgplVersionCompat
 	"
```

Reply:
BTW, these custom variables are handy if/when we create the [debuginfo](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#debuginfo-package) package. For this recipe we could insert this block before the `BUILD()` function:
```diff
 	cmd:make
 	cmd:sed
 	"
+
+defineDebugInfoPackage libatomic_ops$secondaryArchSuffix \
+        "$libDir"/libatomic_ops.so.$libopsVersion \
+        "$libDir"/libatomic_ops_gpl.so.$libgplVersion
+
 BUILD()
 {
```

Does anyone know if, for this particular recipe, it would be a good idea to create the debuginfo package?
(Those contributors who think it's a good idea, please add a 👍 to this message. Those who think the opposite, please add a 👎 , of course. Thanks!)
(Of course, if anyone wishes to write a real comment, please do so!)